shopfloor/services/checkout.py

```diff
@@ -564,35 +564,37 @@ def _select_lines_from_lot(
                     "body": _("Lot is not in the current transfer."),
                 },
             )
-
-        # When lots are as units outside of packages, we can select them for
-        # packing, but if they are in a package, we want the user to scan the packages.
-        # If the product is only in one package though, scanning the lot selects
-        # the package.
```

Comment:
I'm not sure to understand in what this comment was wrong?

Reply:
It was not wrong, I thought it was clearer to have the explanation on each `if` cases

Reply:
Both is good IMO, to get the big picture before entering in the "if/else" dance, and each block gets its own comment

Reply:
It was to get the big picture before reading all the code details. But OK